starter-code/Portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/stayle.css">
+    <title>Lamees AlFallaj |Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <a target="_blank" href="index.html">About me</a> |
+            <a href="#">Portfolio</a>
+        </nav>
+        <h1>LAMEES ALFALLAJ</h1>
+    </header>
+    <main>
+        <div class="content portfolio">
+            <h2>Experience</h1>
+                <p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling,
+                    and spending time with my goldfish, Jerry.
+                </p>
+                <h2>Relevant Experience</h2>
+                <ul>
+                    <li>Web Developer, Relaxr 2015 - present</li>
+                    <p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices. </p>
+                    <img src="images/relaxr-deliverable.png" alt="Relaxr">
+                    <li>Web Developer, Startup Matchmaker 2015 - present</li>
+                    <p>Used a design team's wireframes to develop this company's responsive homepage. </p>
+                    <img src="images/startup-matchmaker-deliverable.png" alt="startup-matchmaker-deliverable">
+                    <li>Web Developer, CitiPix 2015 – present</li>
+                    <p>Built web app prototype that allows users to store and quickly retrieve photos using keywords. </p>
+                    <img src="images/citipix-deliverable.png" alt="citipix-deliverable">
+                </ul>
+                <h2>EDUCATION</h2>
+                <ul>
+                    <li>General Assembly — FEWD 2015</li>
+                    <li>College of Soft Knocks — B.S. 2009</li>
+                    <li>School of Hard Knocks — 2005 (Honors)</li>
+                </ul>
+
+        </div>
+    </main>
+</body>
+
+<footer>
+    <a href="www.facebook.com" target="_blank">Facebook</a> |
+    <a href="www.twitter.com" target="_blank">Twitter</a> |
+    <a href="www.linkedin.com" target="_blank">Linkedin</a> |
+    <a href="www.instagram.com" target="_blank">Instagram</a>
+</footer>

```

Comment:
It's a good idea to test your links and see if they work. I've tried to go to the links and they haven't worked... not sure if they worked with you but if you include "http://" at the beginning of the URL it will work. Try it :+1: